spec/rubocop/cop/rails/file_path_spec.rb

```diff
@@ -122,6 +122,14 @@
         RUBY
       end
     end
+
+    context 'when concat Rails.root and colon using string interpoloation' do
+      it 'does not register an offense' do
+        expect_no_offenses(<<~'RUBY')
+          "#{Rails.root}:/foo/bar"
```

Comment:
Interesting. What platform will this path representation be used on?

Reply:
If my understanding is correct, path lists separated by colons are usually used in UNIX world. For example, `$PATH` is one of them.

```
root@2d99b0868caf:/# echo $PATH
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

In another case, `-v` option of `docker` command takes a pair of paths like `-v /path/to/local:/path/to/container`. The real case I faced is this.

```
# in Rakefile
sh "docker run -v #{Rails.root}:/app blah blah blah"
```